packages/cli/lib/utils.js

```diff
@@ -17,13 +17,15 @@ const _ = require('lodash');
 const pascalCase = require('change-case').pascalCase;
 const lowerCase = require('change-case').lowerCase;
 const promisify = require('util').promisify;
-const camelCase = require('change-case').camelCase;
+const utilCamelCase = require('change-case').camelCase;
 const pluralize = require('pluralize');
 const urlSlug = require('url-slug');
 const validate = require('validate-npm-package-name');
 const Conflicter = require('yeoman-generator/lib/util/conflicter');
 const connectors = require('../generators/datasource/connectors.json');
 const stringifyObject = require('stringify-object');
+const camelCase = require('camelcase');
```

Comment:
It's very confusing to have two `camelCase`? What's the difference between `camelCase` and `change-case.camelCase`?

Reply:
I believe that `lodash` already supports `camelCase`.

Reply:
I found out that `camelCase` is a smaller package extracted from `change-case`. So `camelCase` and `change-case.camelCase` is the same function.

Reply:
But when I tested it yesterday, ```change-case.camelCase``` keeps the underscores while ```camelCase``` doesn't. And we use ```change-case.camelCase``` in ```this.artifactInfo.modelVariableName```, so I kept it and renamed it.

Reply:
> I tested it yesterday, change-case.camelCase keeps the underscores while camelCase doesn't.

I think it's the other way around:
- change-case.camelCase does not use `_`
- camelCase uses `_`

Reply:
Just tested it out. I think we can get rid of the new imported camelCase since``` _.camelCase``` does the same thing. 
```
? Model class name: to1doList_todo
NAME: to1doList_todo
change-case.camel: to1doListTodo
New imported camel: to1DoListTodo
Lodash camle :to1DoListTodo
```
```
? Model class name: to_1doList_todo
NAME: to_1doList_todo
change-case.camel: to_1doListTodo
New imported camel: to1DoListTodo
Lodash camle :to1DoListTodo
```
But there still a slight difference between ```change-case.camel``` and ```_.camel```. I think we should use ```_.camel``` as we use pascal case to convert it to class name.

Reply:
Yes.